app/controllers/api/storage.php

```diff
@@ -944,9 +944,11 @@
         $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $project->getId() . DIRECTORY_SEPARATOR . $bucketId . DIRECTORY_SEPARATOR . $fileId)); // Limit file number or size
         $data = $cache->load($key, 60 * 60 * 24 * 30 * 3/* 3 months */);
 
-        if ($data) {
-            $output = (empty($output)) ? $type : $output;
+        if(empty($output)) {
+            $output = empty($type) ? (array_search($mime, $outputs) ?? 'jpg') : $type;
```

Comment:
If reading correctly, we are falling back to jpg for file type?

Reply:
yes at this point, what we receive will be image. Non image files will use default logo images for preview.